src/react-demo/Content/src/Hmr.fs

```diff
@@ -0,0 +1,71 @@
+module Hmr
+
+open Elmish
+open Elmish.Browser.Navigation
+open Elmish.Browser.UrlParser
+open Fable.Core
+open Fable.Core.JsInterop
+open Fable.Import
+open Fable.Import.Browser
+open App.State
+open Fable.Helpers.React
+open Fable.Helpers.React.Props
+
+module Program =
+
+    type IModule =
+        abstract hot: obj with get, set
+
+    let [<Global>] [<Emit("module")>] ``module`` : IModule = jsNative
+
+    type HMRMsg<'msg> =
+        | UserMsg of 'msg
+        | Reload
+
+    type HMRModel<'model> =
+        { HMRCount : int
+          UserModel : 'model }
+
+    let mutable hmrState : obj = null
+
+    let inline withHMR (program:Elmish.Program<'arg, 'model, 'msg, 'view>) =
```

Comment:
inline?

Reply:
Yes, we need to have ` ``module``.hot?accept() |> ignore` called from the entry file of the application.

Reply:
I see